### PR TITLE
Add BUCKET_INTERNAL_INGEST as a container parameter

### LINKED ISSUE #102: Add `BUCKET_INTERNAL_INGESTION` for initializing data
Currently, the admin server places data into `BUCKET_INTERNAL_PRIVATE`, which is also used for intermediate values. This can lead to the administrator (the server that handles ingestion/partitioning data) to have overly broad access to each of the server buckets. In the Origin Telemetry case, server A and the admin server are both run in Airflow with access to the same set of individuals.

### PR DESCRIPTION
**Pull request by acmiyaguchi**
This fixes #102 and allows for cleaner semantics between the use of each bucket.

### CODE REVIEW

**Comment by acmiyaguchi** [comment]
A lot of this is duplicated from #113, with `admin` now being referred to as `ingest` and the addition of a new bucket to separate concerns.

File: deployment/testing-v4/terraform/main.tf [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+// testing whether origin telemetry inserts into BigQuery correctly
+
+// The ingest container will be used for coordination, and gets access to
+// server A's private bucket because they are operated by the same entity.
+resource "google_storage_bucket_iam_member" "ingest_internal_private" {
+  bucket = module.bucket-a.private
+  role   = "roles/storage.objectAdmin"
+  member = "serviceAccount:${google_service_account.ingest.email}"
+}
```
Comment (acmiyaguchi) [inline deployment/testing-v4/terraform/main.tf:92]: In my opinion, this is the ugliest part of this configuration. However, this makes the relationship explicit. The ingest server also has more permissions than would be specified here, like access to shared-prod and the like.

**Review by wlach — CHANGES_REQUESTED** [review]
I *think* I understand what you're doing here, but it's not 100% clear. Shouldn't you update  https://github.com/mozilla/prio-processor/blob/main/docs/guide.md as well? Doing so will have the side benefit of making this PR easier to review.

**Comment by acmiyaguchi** [comment]
I've updated the documentation to reflect the new variables. Some of those sections will require a little bit of rework, but I'll do my best to update the docs to reflect any changes I'm making now. It's nice to have a skeleton to work on.

**Review by wlach — APPROVED** [review]

**Pull request merged**